projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/received-issue/received-issue.component.ts

```diff
@@ -15,32 +15,39 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component, EventEmitter, Input, Output } from '@angular/core';
-import { TranslateService } from '@ngx-translate/core';
-import { RecordUiService } from '@rero/ng-core';
-import { IssueItemStatus } from '@rero/shared';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { HoldingsService } from '@app/admin/service/holdings.service';
+import { IssueService } from '@app/admin/service/issue.service';
 import { RecordPermissionService } from '@app/admin/service/record-permission.service';
+import { TranslateService } from '@ngx-translate/core';
+import { RecordUiService } from '@rero/ng-core';
+import { IssueItemStatus, UserService } from '@rero/shared';
 
 @Component({
   selector: 'admin-received-issue',
   templateUrl: './received-issue.component.html',
-  styleUrls: ['../serial-holding-detail-view.style.scss']
+  styleUrls: ['../serial-holding-detail-view.style.scss'],
+  providers: [IssueService]
```

Comment:
why this service need to be into `providers` ?

Reply:
This is a proposal from @jma. Load the service only on the component and not on the root.